src/main/scala/mesosphere/marathon/core/task/tracker/impl/InstanceTrackerDelegate.scala

```diff
@@ -97,14 +97,16 @@ private[tracker] class InstanceTrackerDelegate(
         val effectF = (instanceTrackerRef ? update)
           .mapTo[InstanceUpdateEffect]
           .transform {
-            case s@Success(_) => logger.info(s"Completed processing instance update ${update.operation.shortString}"); s
-            case f@Failure(e: AskTimeoutException) => logger.error(s"Timed out waiting for response for update $update", e); f
-            case f@Failure(t: Throwable) => logger.error(s"An unexpected error occurred during update processing of: $update", t); f
+            case s @ Success(_) =>
```

Comment:
I don't like those new lines. Breaks the code alignment aesthetics :P

Reply:
That's what sbt or IntelliJ is doing, dude. I'll revert it except that I prefer the spaces.

Reply:
You allow your Intellij too much intellijence 🥁-tsss...